### DISCUSSION TITLE
Tuple unpacking is giving me "type is unknown" message

### DISCUSSION THREAD

**Discussion by paxcodes** [microsoft/pyright#1451] [Q&A] [answered]
Why is the following code giving me "type is unknown" messages

```python
from _pytest.fixtures import SubRequest

@fixture(scope="module")
def someFixture1(request: SubRequest):
    rssUrl: str
    daysToRetrieve: int
    rssUrl, daysToRetrieve = request.param # Getting an error here, `Type of "rssUrl" is unknown` and `Type of "daysToRetrieve" is unknown`
    ...
```

But code like this is OK,

```python
@fixture(scope="module")
def someFixture2(request: SubRequest):
    rssUrl: str = request.param
    ...
```

**Comment by erictraut** [comment]
Are you using strict mode? You should not receive "type is unknown" unless `reportUnknownVariableType` is enabled. It's off by default unless you enable it manually or enable strict mode.

I receive "type is unknown" errors in both of the examples you provided.

![Screen Shot 2021-02-04 at 2 55 42 PM](https://user-images.githubusercontent.com/7040122/106965820-0f07ee80-66f9-11eb-82ce-36269d5f0452.png)

**Comment by erictraut** [comment, marked as answer]
Looks like there's a missing annotation in the pytest code.

![image](https://user-images.githubusercontent.com/7040122/106966124-9190ae00-66f9-11eb-80fe-db319cb2f144.png)
